kmsauth/__init__.py

```diff
@@ -175,6 +195,7 @@ def decrypt_token(self, username, token):
                 }
                 if version > 1:
                     context['user_type'] = user_type
+                context.update(self.extra_context)
```

Comment:
Do you want to do this in the reverse order (`extra_context.update(context)`) so that `extra_context` keys can never clobber `context` keys?